mongo/integration/unified/session_options.go

```diff
@@ -89,10 +88,6 @@ func (so *sessionOptions) UnmarshalBSON(data []byte) error {
 	if temp.Causal != nil {
 		so.SetCausalConsistency(*temp.Causal)
 	}
-	if temp.MaxCommitTimeMS != nil {
```

Comment:
`maxCommitTimeMS` appears in `defaultTransactionOptions` in the unified test format, not in `sessionOptions`.